xe/do.py

```diff
@@ -33,6 +36,6 @@ def run(cmd):
 
     print('Running: %s' % cmd)
     try:
-        call(cmd)
+        call(cmd, shell=(os.name == 'nt'))
```

Comment:
So we should do two things here. First create a use_shell function to encapsulate that logic. Second, if we do use the shell, I think we'll need to join the cmd list to a string. If you try that out and find otherwise or if a list is OK on windows with shell=True but not OK with POSIX, then we might have to do something more involved.